devito/compiler.py

```diff
@@ -86,7 +78,6 @@ def __init__(self, *args, **kwargs):
 
         if self.openmp:
             self.ldflags += ['-fopenmp']
-        self.pragma_ivdep = [Pragma('GCC ivdep')]
```

Comment:
This comment only stands for this ``GCC ivdep``

The other pragmas now are part of the "compiler language" under the DLE (grep "Pragma" in /dle for more info)